### DISCUSSION TITLE
CORS error without vpn

### DISCUSSION THREAD

**Discussion by Andrew-Marty00shenko** [WalletConnect/walletconnect-monorepo#3842] [Developer Support] [answered]
Hey guys, ive a problem, without vpn i get a promblem cors

**Comment by glitch-txs** [comment]
Hi, could you provide more information?
- What library and versions are you using?
- where are you based?
- could you share your project ID or email account?

**Reply by Andrew-Marty00shenko** [reply]
"dependencies": {
    "@emotion/react": "^11.11.1",
    "@mantine/core": "^6.0.19",
    "@mantine/hooks": "^6.0.19",
    "@mantine/modals": "^6.0.19",
    "@reduxjs/toolkit": "^1.9.5",
    "@web3modal/ethereum": "^2.7.1",
    "@web3modal/react": "^2.7.1",
    "@web3modal/wagmi": "^3.0.2",
    "classnames": "^2.3.2",
    "i18next": "^23.5.1",
    "react": "^18.2.0",
    "react-awesome-reveal": "^4.2.5",
    "react-copy-to-clipboard": "^5.1.0",
    "react-dom": "^18.2.0",
    "react-hook-form": "^7.45.4",
    "react-i18next": "^13.2.2",
    "react-image-webp": "^0.8.0",
    "react-redux": "^8.1.2",
    "react-router-dom": "^6.15.0",
    "react-toastify": "^9.1.3",
    "sass": "^1.66.1",
    "viem": "^1.13.1",
    "wagmi": "^1.4.2"
  },

project id is 4f02ec3b9c4b8d29757b9fe76ec24771

web3modal

**Reply by Andrew-Marty00shenko** [reply]
error  cors
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/56186268/b82030ac-c3f3-4982-a6f1-4a23a8920414)

**Reply by arein** [reply]
Please right click on the failing call to web3modal/walletconnect in the network tab and `copy as cURL`

**Reply by Andrew-Marty00shenko** [reply]
i cant do it. A user from the site had sent this error to me. I dont get errrors from my PC...

**Reply by Andrew-Marty00shenko** [reply]
so you understand, all this worked until today. that is, no one had an error, but now some have it, and some don’t

**Reply by glitch-txs** [reply]
do you have information on where is the user based?

**Reply by Andrew-Marty00shenko** [reply]
yes, from Ukraine

**Reply by Norisens** [reply]
All Ukraine IPs banned, I and my friends have the same error

**Reply by Andrew-Marty00shenko** [reply]
Hm, as ive said this works until today. how?

**Reply by Norisens** [reply]
Everything worked for me until today, but since Wallet Connect blocked all Ukrainian IP addresses, now we do not have access to API requests.

**Reply by Norisens** [reply]
I can't even log into the official Wallet Connect website

**Reply by Norisens** [reply]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/141584545/6f3f1bf5-4cb7-423e-b69b-4848e67dbea2)

**Reply by Andrew-Marty00shenko** [reply]
that's crazy! but why

**Reply by Norisens** [reply]
I'm trying to figure this out too

**Reply by Andrew-Marty00shenko** [reply]
in this case we should rewrite our app. maybe u can give me an alternative of walletconnect

**Reply by Norisens** [reply]
I'm already doing this, https://codesandbox.io/examples/package/web3modal

**Reply by Andrew-Marty00shenko** [reply]
is it resolving the problem? here is walletconnect

**Reply by glitch-txs** [reply]
Thanks for reporting, raising this to the team, we'll try to fix it asap.

**Reply by Andrew-Marty00shenko** [reply]
Did I understand correctly that I can leave the code unchanged? will you fix this bug?

**Reply by Andrew-Marty00shenko** [reply]
I launched it on localhost and I also got a cors error

**Reply by glitch-txs** [reply]
> Did I understand correctly that I can leave the code unchanged

Yes, the issue is on our end

**Reply by Andrew-Marty00shenko** [reply]
Ok. Can you know me when the problem is solved?

**Reply by Andrew-Marty00shenko** [reply]
@glitch-txs can you say approximately when the problem will be solved?

**Reply by HarlamovEvgeniy** [reply]
@glitch-txs Hi, tell me, is this a similar problem for users from Russia? Or is there another reason for blocking them?

**Reply by meri-maki** [reply]
Getting reports from users from Russia and Ukraine - web3modal not showing wallets on mobile devices
Would be great to get an estimated time of the fix... to see if we should rewrite everything

packages for ref
```
"@web3modal/wagmi": "^3.1.0",
"viem": "^1.17.2",
"wagmi": "^1.4.5"
```

**Reply by Norisens** [reply]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/141584545/f361209c-7c2b-4cba-9ce6-001a76811845)
The developers seem to be kidding me

**Reply by glitch-txs** [reply]
we'll share an announcement shortly

**Reply by chris13524** [reply]
@Norisens your screenshot looks like a phishing attempt, 0xYachi is not associated with WalletConnect.

**Reply by Norisens** [reply]
Yes, I already realized that, he sent me a site to validate my wallet and it’s a phishing link
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/141584545/dda407e5-9b7c-4c8c-ae32-0d247c679f44)

**Reply by Norisens** [reply]
The group is also fake

**Reply by chris13524** [reply]
@Norisens yes the group is fake, please report it

**Reply by gymnasy55** [reply]
Everything seems to be working fine, thx to the team.

**Reply by meri-maki** [reply]
https://github.com/WalletConnect/blockchain-api/pull/336

Well, what a shame

**Comment by glitch-txs** [comment, marked as answer]
Please check
https://github.com/orgs/WalletConnect/discussions/3862
